nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -0,0 +1,237 @@
+"""
+Transformer for computing seeds signals.
```

Comment:
Might not strike as a crisp description. Perhaps "Mask nifti images by spherical volumes for seed-region analyses"